anchor.js

```diff
@@ -244,8 +244,7 @@
 
       // Regex for finding the non-safe URL characters (many need escaping):
       //   & +$,:;=?@"#{}|^~[`%!'<>]./()*\ (newlines, tabs, backspace, vertical tabs, and non-breaking space)
-      var nonsafeChars = /[& +$,:;=?@"#{}|^~[`%!'<>\]./()*\\\n\t\b\v\u00A0]/g,
-          urlText;
```

Comment:
I can revert the temp var removal; I thought it helped with uglify but apparently it doesn't. So, if it's more clear for you with the temp var, let me know and I'll revert it.

Reply:
I think it's good without the temp variable. 👍